<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>superPage1</class>
 <widget class="QWidget" name="superPage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>管理房间信息</string>
      </attribute>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>691</width>
         <height>511</height>
        </rect>
       </property>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>641</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>641</width>
          <height>60</height>
         </rect>
        </property>
        <property name="title">
         <string>房间管理</string>
        </property>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>180</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2">
          <item>
           <widget class="QPushButton" name="last_button">
            <property name="text">
             <string>上一页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="next_button">
            <property name="text">
             <string>下一页</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>30</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>我是标签</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>10</y>
           <width>231</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="placeholderText">
             <string>请输入要查询的房间号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="findroom_Button">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>添加房间信息</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>291</width>
         <height>381</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="widget_8" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>251</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>  房间号：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="roomid_lineEdit">
              <property name="placeholderText">
               <string>请输入房间号</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>房间价格：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="roomprice_lineEdit">
              <property name="placeholderText">
               <string>请输入价格</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>54</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>房间类型：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="roomtype_comboBox">
              <item>
               <property name="text">
                <string>单人间</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>双人房</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>家庭房</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>总统套房</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1">
            <property name="rightMargin">
             <number>65</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>房间状态：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="roomstat_comboBox">
              <item>
               <property name="text">
                <string>空闲中</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>租用中</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>已预订</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_9" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="pic_pushButton">
              <property name="text">
               <string>  上传照片</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="placeholderText">
               <string>显示照片路径</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="add_pushButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
